### PR TITLE
calibre: 3.3.0 -> 3.5.0

### PR DESCRIPTION
**Pull request by jfrankenau**
###### Motivation for this change

Update.

Calibre now uses the new html5-parser package instead of html5lib. I have also removed some unused dependencies and installed the man pages.

###### Things done

Please check what applies. Note that these are not hard requirements but mereley serve as information for reviewers.

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---
cc @viric @domenkozar @pSub @AndersonTorres

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -2352,6 +2352,28 @@ in {
     };
   };
 
+  html5-parser = buildPythonPackage rec {
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: See the header of this file. Please move to a separate expression.

File: pkgs/applications/misc/calibre/default.nix [hunk 2]
```diff
@@ -36,7 +36,7 @@ stdenv.mkDerivation rec {
 
     # Remove unneeded files and libs
     rm -rf resources/calibre-portable.* \
-           src/{chardet,cherrypy,html5lib,odf,routes}
```
Comment (FRidh) [inline pkgs/applications/misc/calibre/default.nix]: are these not vendored any more that you remove these items?

**Review by FRidh — CHANGES_REQUESTED** [review]
Looks good to me. The expression needs to be moved and then the impact of the `lxml` bump would have to be tested.

File: pkgs/top-level/python-packages.nix (hunk 1 above)
Comment (jfrankenau) [inline pkgs/top-level/python-packages.nix]: Done.

File: pkgs/applications/misc/calibre/default.nix (hunk 2 above)
Comment (jfrankenau) [inline pkgs/applications/misc/calibre/default.nix]: Yes, these are not bundled with Calibre anymore.

**Comment by FRidh** [comment]
Is the `lxml` bump necessary?

**Comment by jfrankenau** [comment]
Yes, it is. I've been playing around with `nox-review` to test the version bump of `lxml` but it seems to fail due to something else. Still need some time evaluating it.

**Comment by vcunat** [comment]
@FRidh: ping?  (you self-assigned this)

**Comment by FRidh** [comment]
Testing the Python packages that need to be rebuild at https://headcounter.org/hydra/eval/369772. Other rebuilds I cannot really test.

**Comment by FRidh** [comment]
I've pushed these commits to staging.

**Pull request closed without merging**